src/utils/toast.ts

```diff
@@ -0,0 +1,23 @@
+import { toast, setDefaults } from 'bulma-toast';
+
+export function showToast(err, options = {}) {
+  // try to get a string from the error object
+  const message = err.message || err.toString();
+
+  // options will be merged with these and the defaults
+  // will be used if the fields are not provided.
+  setDefaults({
+    closeOnClick: true,
+    dismissible: true,
+    duration: 2000,
+    opacity: 1,
+    position: 'top-center',
```

Comment:
```suggestion
    position: 'top-right',
```

Reply:
It will cover the connect button. If it's ok by you, it's ok by me.

Reply:
I see, but I think is fine, is not used much after first enable, and toast it will disappear after 2 secs